parsec/core/logged_core.py

```diff
@@ -37,7 +37,7 @@
     RemoteDevicesManagerBackendOfflineError,
     RemoteDevicesManagerNotFoundError,
 )
-from parsec.core.mountpoint import mountpoint_manager_factory
+from parsec.core.mountpoint import mountpoint_manager_factory, MountpointManager
```

Comment:
`MountpointManager` is exposed in `parsec.core.mountpoint`

Reply:
I don't understand you point...

Reply:
(sorry, i made a typo) :
`MountpointManager` can not be imported directly from `parsec.core.mountpoint` but from `parsec.core.mountpoint.manager`

I fixed it:
https://github.com/Scille/parsec-cloud/pull/1415/commits/2f58d31a9a861e83b23c70472fd86c1f6c8a1906